test/Microsoft.AspNet.Razor.Test/TestFiles/CodeGenerator/CS/Source/MinimizedTagHelpers.cshtml

```diff
@@ -0,0 +1,21 @@
+﻿@addTagHelper "something, nice"
+
+<p catchall-unbound-required>
+    <input nottaghelper />
+    <input class="btn" catchall-unbound-required />
+    <input class="btn" catchall-unbound-required input-unbound-required input-bound-required-string="hello" />
+    <input class="btn" catchall-unbound-required input-unbound-required catchall-bound-string="world" input-bound-required-string="hello2" />
+    <input class="btn" catchall-unbound-required="hello" input-unbound-required="hello2" catchall-unbound-required input-bound-required-string="world" />
+    <input nottaghelper
+           class="btn"
+           catchall-unbound-required
+           input-unbound-required
+           bound-string="hello" />
+    <input 
+           nottaghelper
+           class="btn"
+           catchall-unbound-required
+           input-unbound-required
+           input-bound-required-string="hello"
+           input-unbound-required="world" />
```

Comment:
Mix whitespace into the first four cases rather than duplicating a couple.  Don't duplicate even more tests.